src/main/java/orangehrm/OrangeRecruitmentPage.java

```diff
@@ -0,0 +1,246 @@
+
+    @FindBy(id = "addCandidate_appliedDate")
+    private WebElement appliedDate;
+
+    @FindBy(id = "addCandidate_consentToKeepData")
+    private WebElement consentDataCheckbox;
+
+    public OrangeRecruitmentPage(WebDriver driver){
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    public WebElement getCreatedVacancy() {
+        return createdVacancy;
+    }
+
+    public WebElement getCreatedCandidate() {
+        return createdCandidate;
+    }
+
+    public OrangeRecruitmentPage toVacancies(){
+        vacancies.click();
+        return this;
+    }
+
+    public OrangeRecruitmentPage clickAddBtn(){
+        addButton.click();
+        return this;
+    }
+
+    public OrangeRecruitmentPage selectJobTitle(){
+        Select jobTitle = new Select(jobTitleList);
+        jobTitle.selectByIndex(1);
+        title = jobTitle.getFirstSelectedOption().getText();
+        return this;
+    }
+
+    public OrangeRecruitmentPage setVacancyName(String nameVacancy){
+        vacancyName.sendKeys(nameVacancy);
+        recordVacancyName = nameVacancy;
+        return this;
+    }
+
+    public OrangeRecruitmentPage setHiringManager(String nameManager){
+        hiringManager.sendKeys(nameManager);
+        return this;
+    }
+
+    public OrangeRecruitmentPage setPositionsNumber(){
+        numberOfPositions.sendKeys("1");
+        return this;
+    }
+
+    public OrangeRecruitmentPage setDescription(String text){
+        description.sendKeys(text);
+        return this;
+    }
+
+    public OrangeRecruitmentPage uncheckRssFeed(){
+        feedCheckbox.click();
+        return this;
+    }
+
+    public OrangeRecruitmentPage clickSaveBtn(){
+        saveBtn.click();
+        WebDriverWait saveWait = new WebDriverWait(driver, 10);
+        saveWait.until(ExpectedConditions.invisibilityOfElementLocated(By.className("message.success.fadable")));
+        return this;
+    }
+
+    public OrangeRecruitmentPage clickBackBtn(){
+        backBtn.click();
+        return this;
+    }
+
+    public OrangeRecruitmentPage createVacancy(String nameVacancy, String nameManager, String text){
+        return clickAddBtn()
+                .selectJobTitle()
+                .setVacancyName(nameVacancy)
+                .setHiringManager(nameManager)
+                .setPositionsNumber()
+                .setDescription(text)
+                .uncheckRssFeed()
+                .clickSaveBtn();
+    }
+
+    public OrangeRecruitmentPage findCreatedVacancy(String nameVacancy, String hiringManager){
+        createdVacancy = resultList.stream()
+                .filter(o -> o.findElements(By.xpath("./td")).get(1).getText().contains(nameVacancy) &&
+                        o.findElements(By.xpath("./td")).get(2).getText().contains(title) &&
+                        o.findElements(By.xpath("./td")).get(3).getText().contains(hiringManager))
+                .findFirst().orElse(null);
+        return this;
+    }
```

Comment:
There are a lot of duplication of code like `(By.xpath("./td")`.

Reply:
to specify what exactly text should be placed in what exactly 'td' tag
fixed